app/javascript/Components/Helpers/table_helpers.jsx

```diff
@@ -1,10 +1,34 @@
 import React from "react";
+import {Grid} from "react-loader-spinner";
 
 /**
  * @file
  * Provides generic helper functions and components for react-table tables.
  */
 
+export function customLoadingProp(props) {
+  const {loading} = props;
```

Comment:
You can simplify this code by using the same syntax as in other functions below:

```python
export function customLoadingProp({loading}) {
    ...
```